agentstack/generation/gen_utils.py

```diff
@@ -66,3 +71,72 @@ def string_in_file(file_path: str, str_to_match: str) -> bool:
         file_content = file.read()
         return str_to_match in file_content
 
+
+def _framework_filename(framework: str, path: str = ''):
+    if framework == 'crewai':
+        return f'{path}src/crew.py'
+
+    print(term_color(f'Unknown framework: {framework}', 'red'))
+    sys.exit(1)
+
+
+class CrewComponentType(str, Enum):
```

Comment:
Enum is perfect but whenever I see how often it's referenced I always regret that I didn't make the name shorter. 

`CrewComponent`?

Reply:
agreed, i can do this